CONTRIBUTING.md

```diff
@@ -1,41 +1,78 @@
-# Contributing to eXist
-We welcome all contributions to eXist!
+# Contributing to eXist-db
+We welcome everyone to contribute to eXist-db. We will consider each individual contribution on its own merits.
+We strongly suggest that you join the [eXist-db Slack Channel](https://exist-db.slack.com), so that you can collaborate with the eXist-db community. It is often valuable to discuss a potential contribution before undertaking any work.
 
-We strongly suggest that you join the [eXist-development mailing](https://lists.sourceforge.net/lists/listinfo/exist-development "eXist Development Mailing List") list and also subscribe to the [eXist-commits mailing list](https://lists.sourceforge.net/lists/listinfo/exist-commits "eXist SCM Commits Mailing List"), so that you can collaborate with the eXist team and be kept up to date with changes to the codebase.
+We follow a "Hub and Spoke" like development model, therefore you should fork our eXist-db repository, work on branches within your own fork, and then send Pull-Requests for your branches to our GitHub repository.
 
-## General Issues
-eXist uses the [GitFlow](http://nvie.com/git-model) branching model for development. Specifically, we're using the [AVH Edition of GitFlow tools](https://github.com/petervanderdoes/gitflow) version.
+## Branch Naming
+eXist-db uses a [GitFlow](http://nvie.com/git-model) like branching model for development.
```

Comment:
s/ like/-like

Reply:
Thanks @joewiz I pushed a commit to address this.